proposals/kubeflow-steering-committee-charter.md

```diff
@@ -0,0 +1,148 @@
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
+Member</th>
+      <th><br>
+Organization</th>
+      <th><br>
+Github Profile</th>
+      <th><br>
+Term Start</th>
+      <th><br>
+Term End</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Emeritus Committee Members
+
+[This section will be populated when there are retired committee members.]
+
+## Decision process
+
+The steering committee desires to always reach consensus.
+
+### Normal decision process
+
+Decisions requiring a vote include:
+- Issuing written policy
+- Amending existing written policy
+- Creating, removing, or modifying a working group
+- All spending, hiring, and contracting decisions
+- Official responses to publicly raised issues
+- Any other decisions that at least half of the members (rounded down) present decide require a vote
+
+Decisions are made in meetings when a quorum of the members are present and may pass with at least half the members (rounded up) of the committee supporting it.
+
+Quorum is considered reached when at least half of the members (rounded up) are present.  
+Members of KSC may abstain from a vote. Abstaining members will only be considered as contributing to quorum, in the event that a vote is called in a meeting.
+
+### Special decision process
+
+Issues that impacts the KSC governance requires a special decision process. Issues include:
+- Changes to the KSC charter
+- KSC voting rules
+
+The issue may pass with 70% of the members (rounded up) of the committee supporting it.
+
+### Results
+
+The results of the decision process are recorded and made publicly available, unless they pertain to sensitive or previleged matters. The results will include:
+- Description of the issue
+- Names of members who supported, opposed, and abstained from the vote.
+-  
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the steering committee at [TBD]. This is a private discussion list to which all members of the committee have access.
+1. Open an issue on a kubeflow/community repository and indicate that you would like attention from the steering committee using github tag [TBD].
+
+## Changes to the charter
+
+Changes to the KSC charter may be proposed via a PR on the charter itself. Amendments are accepted following the [Special Decision Process](#special-decision-process) detailed above.
```

Comment:
Should we explicitly say that Kubeflow community members can propose changes and improvements for the KSC charter ?

Reply:
I will add this.